src/lev-report/charts/CumulativeUsage.js

```diff
@@ -19,7 +19,7 @@ const CumulativeUsage = ({ data }) => data && data.length &&
 			tickFormat={tick => {
 				const date = moment.unix(tick);
 				const month = date.month();
-				return month ? month % 3 === 0 ? `Q${date.quarter()}` : '' : `'${date.format('YY')}`;
+				return month ? month % 3 === 0 ? `Q${date.quarter()}` : '' : date.format('YYYY');
```

Comment:
Would be nice to have some parens here!

I know the lack of parens wasn't caused by this PR though so it's up to you.

Reply:
On a more relevant subject, why move away from `'20`? - I'd have thought that's more concise and easy to read.

Reply:
Or maybe we should add the full year but remove the quarters...

Reply:
Brackets added for clarity.
Full years was an old request that came from Nigel.

Reply:
Okay.

It will be nice when we've got Chromatic in place and we can do proper visual reviews of things.